<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="font">
          <font>
           <family>Cascadia Code</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbChoice">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="listView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lbName">
         <property name="font">
          <font>
           <family>Cascadia Code</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Имя файла:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="setName">
         <property name="font">
          <font>
           <family>Cascadia Code</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbDepth">
         <property name="font">
          <font>
           <family>Cascadia Code</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Глубина цвета:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="setDepth">
         <property name="font">
          <font>
           <family>Cascadia Code</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lbSize">
         <property name="font">
          <font>
           <family>Cascadia Code</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Размер:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="setSize">
         <property name="font">
          <font>
           <family>Cascadia Code</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbCompr">
         <property name="font">
          <font>
           <family>Cascadia Code</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Сжатие:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="setCom">
         <property name="font">
          <font>
           <family>Cascadia Code</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="lbResolution">
         <property name="font">
          <font>
           <family>Cascadia Code</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Расширение:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="setRes">
         <property name="font">
          <font>
           <family>Cascadia Code</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
